modules/oaeservice/manifests/hilary.pp

```diff
@@ -40,6 +40,7 @@
   $web_domain = hiera('web_domain')
   $app_admin_tenant = hiera('app_admin_tenant', 'admin')
   $admin_domain = "${app_admin_tenant}.${web_domain}"
+  $guest_domain = "guest.${web_domain}"
```

Comment:
No, you can just update it in the tenant admin UI after it has been auto-created